src/main/webapp/js/finscholar/collegepageview/collegepageview.js

```diff
@@ -0,0 +1,53 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/** @fileoverview Base class for College View. */
+
+goog.module('finscholar.collegepageview');
+
+const GoogDom = goog.require('goog.dom');
+const {loadCollegeData} = goog.require('datahandlers.collegepage');
+
+/**
+ * Class for the college page view.
+ * @public
+ */
+class CollegePageView {
+  /**
+   * @constructor
+   * @param {Element} element - The DOM element to render the college page to.
+   */
+  constructor(element) {
+    /** @private {Element} - The DOM element where the CollegePageView will be rendered. */
+    this.element_ = element;
```

Comment:
Just want to mention the loading sequence of web component. For our current design, it is possible that the 'element' to contain college has not been loaded when you want to initiate the CollegeView object. 
You can keep it for now, but on my part I just passed in the element whenever the scholarship page is requested.

Reply:
Good idea!